File: Evenement.java
package Entité;

import java.sql.Date;

/**
 *
 * @author lotfi
 */
public class Evenement {
   private int ideven;
   private String titreeven;
    private String descriptioneven;
    private Date datedebut;
    private Date datefin;
    private String lieueven;
    private String invitees;
    private String respensable;
    
    
    public Evenement()
    {
        //Empty Constructor
    }
    public Evenement(String titreeven, String descriptioneven, Date datedebut,Date datefin,String lieueven,String invitees,String respensable) {
   
        this.titreeven = titreeven;
        this.descriptioneven = descriptioneven;
        this.datedebut = datedebut;
        this.datefin = datefin;
        this.lieueven = lieueven;
        this.invitees = invitees;
        this.respensable = respensable;
    }
    
     
    public Evenement(int ideven, String titreeven, String descriptioneven, Date datedebut,Date datefin,String lieueven,String invitees,String respensable) {
        this.ideven = ideven;
        this.titreeven = titreeven;
        this.descriptioneven = descriptioneven;
        this.datedebut = datedebut;
        this.datefin = datefin;
        this.lieueven = lieueven;
        this.invitees = invitees;
        this.respensable = respensable;
    }

    public int getIdeven() {
        return ideven;
    }

    public void setIdeven(int ideven) {
        this.ideven = ideven;
    }

    public String getTitreeven() {
        return titreeven;
    }

    public void setTitreeven(String titreeven) {
        this.titreeven = titreeven;
    }

    public String getDescriptioneven() {
        return descriptioneven;
    }

    public void setDescriptioneven(String descriptioneven) {
        this.descriptioneven = descriptioneven;
    }

    public Date getDatedebut() {
        return datedebut;
    }

    public void setDatedebut(Date datedebut) {
        this.datedebut = datedebut;
    }

    public Date getDatefin() {
        return datefin;
    }

    public void setDatefin(Date datefin) {
        this.datefin = datefin;
    }

    public String getLieueven() {
        return lieueven;
    }

    public void setLieueven(String lieueven) {
        this.lieueven = lieueven;
    }

    public String getInvitees() {
        return invitees;
    }

    public void setInvitees(String invitees) {
        this.invitees = invitees;
    }

    public String getRespensable() {
        return respensable;
    }

    public void setRespensable(String respensable) {
        this.respensable = respensable;
    }

    @Override
    public String toString() {
        return "Evenement{" + "ideven=" + ideven + ", titreeven=" + titreeven + ", descriptioneven=" + descriptioneven + ", datedebut=" + datedebut + ", datefin=" + datefin + ", lieueven=" + lieueven + ", invitees=" + invitees + ", respensable=" + respensable + '}';
    }
    
    
    
    
    
    
    
}

File: CheckUp/src/Entity/Evenement.php
<?php

namespace App\Entity;

use App\Repository\EvenementRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * @ORM\Entity(repositoryClass=EvenementRepository::class)
 */
class Evenement
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut donner un titre")
     */
    private $titreeven;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut donne la description ")
     */
    private $descriptioneven;

    /**
     * @ORM\Column(type="date")
     * @Assert\NotBlank(message="il faut donne une date")
     */
    private $dateFin;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut donne un lieu")
     */
    private $lieuEven;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut donne les invitees")
     */
    private $invites;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut donne un responsable")
     */
    private $responsable;

    /**
     * @ORM\Column(type="date")
     * @Assert\NotBlank(message="il faut donne une date")
     */
    private $dateDebut;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getTitreeven(): ?string
    {
        return $this->titreeven;
    }

    public function setTitreeven(string $titreeven): self
    {
        $this->titreeven = $titreeven;

        return $this;
    }

    public function getDescriptioneven(): ?string
    {
        return $this->descriptioneven;
    }

    public function setDescriptioneven(string $descriptioneven): self
    {
        $this->descriptioneven = $descriptioneven;

        return $this;
    }

    public function getDateFin(): ?\DateTimeInterface
    {
        return $this->dateFin;
    }

    public function setDateFin(\DateTimeInterface $dateFin): self
    {
        $this->dateFin = $dateFin;

        return $this;
    }

    public function getLieuEven(): ?string
    {
        return $this->lieuEven;
    }

    public function setLieuEven(string $lieuEven): self
    {
        $this->lieuEven = $lieuEven;

        return $this;
    }

    public function getInvites(): ?string
    {
        return $this->invites;
    }

    public function setInvites(string $invites): self
    {
        $this->invites = $invites;

        return $this;
    }

    public function getResponsable(): ?string
    {
        return $this->responsable;
    }

    public function setResponsable(string $responsable): self
    {
        $this->responsable = $responsable;

        return $this;
    }

    public function getDateDebut(): ?\DateTimeInterface
    {
        return $this->dateDebut;
    }

    public function setDateDebut(\DateTimeInterface $dateDebut): self
    {
        $this->dateDebut = $dateDebut;

        return $this;
    }
}
